x-pack/solutions/security/plugins/security_solution/public/entity_analytics/components/privileged_user_monitoring_onboarding/add_data_source.tsx

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React, { useCallback, useState } from 'react';
+import {
+  EuiCard,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiHorizontalRule,
+  EuiIcon,
+  EuiPanel,
+  EuiSpacer,
+  EuiText,
+  EuiTitle,
+} from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { useIntegrationLinkState } from '../../../common/hooks/integrations/use_integration_link_state';
+import { addPathParamToUrl } from '../../../common/utils/integrations';
+import { ENTITY_ANALYTICS_PRIVILEGED_USER_MONITORING_PATH } from '../../../../common/constants';
+import { useNavigation } from '../../../common/lib/kibana';
+import { INTEGRATION_APP_ID } from '../../../onboarding/components/onboarding_body/cards/integrations/constants';
+import { IndexSelectorModal } from './select_index_modal';
+import { PRIVILEGED_USER_MONITORING_INTEGRATIONS } from './constants';
+
+export const AddDataSourcePanel = () => {
+  const { navigateTo } = useNavigation();
+  const state = useIntegrationLinkState(ENTITY_ANALYTICS_PRIVILEGED_USER_MONITORING_PATH);
+
+  const navigateToIntegration = useCallback(
+    (id: string) => {
+      navigateTo({
+        appId: INTEGRATION_APP_ID,
+        path: addPathParamToUrl(
+          `/detail/${id}/overview`,
+          ENTITY_ANALYTICS_PRIVILEGED_USER_MONITORING_PATH
+        ),
+        state,
+      });
+    },
+    [navigateTo, state]
+  );
+
+  const [isIndexModalOpen, setIsIndexModalOpen] = useState(false);
+
+  return (
+    <EuiPanel paddingSize="xl" hasShadow={false} hasBorder={false}>
+      <EuiTitle>
+        <h3>
+          <FormattedMessage
+            id="xpack.securitySolution.entityAnalytics.privilegedUserMonitoring.addDataSource.title"
+            defaultMessage="Add data source of your privileged users"
+          />
+        </h3>
+      </EuiTitle>
+      <EuiSpacer size="m" />
+      <EuiText size="m">
+        <p>
+          <FormattedMessage
+            id="xpack.securitySolution.entityAnalytics.privilegedUserMonitoring.addDataSource.description"
+            defaultMessage="To get started, define your privileged users by adding an integration with your organization’s user identities, select an index with the relevant data, or import your list of privileged users from a CSV file."
+          />
+        </p>
+      </EuiText>
+
+      <EuiSpacer size="xl" />
+
+      <EuiFlexGroup direction="row" justifyContent="spaceBetween">
+        {PRIVILEGED_USER_MONITORING_INTEGRATIONS.map((integration) => (
+          <EuiFlexItem grow={1} key={integration.id}>
+            <EuiCard
```

Comment:
Could we use the component from Fleet for this? Seems we already do in some places in Secsol e.g

https://github.com/elastic/kibana/blob/da25d13a2ac5be97216e202bf5d6bda6ff920b3b/x-pack/solutions/security/plugins/security_solution/public/onboarding/components/onboarding_body/cards/integrations/integrations_card.tsx#L13

Reply:
After several attempts, I finally succeeded by exporting `useGetPackageInfoByKeyQuery` and `PackageCard` from the `fleet` package.